openelevationservice/server/api/querybuilder.py

```diff
@@ -225,6 +297,77 @@ def polygon_elevation(geometry, format_out, dataset):
                            'The requested geometry is outside the bounds of {}'.format(dataset))
         
     return result_geom
+##Fin-Original code for the polygon_elevation function
```

Comment:
It is not necessary to comment on where a function ends. Fin? Spanish?

Reply:
Done. Deleted.